### PR TITLE
3127 feat/meter import count

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
After importing meters from a property file there is a display describing which meters and their properties were imported. There is no indicator showing how many meters or properties containing meters were imported. This same flow is repeated for the GreenButton upload.

#### What's this PR do?
Adds property and meter counts to the data upload modal during the meter import and GreenButton upload steps.

#### How should this be manually tested?
Test Property Upload:
- Import a Property file with meter data. Following the property import click "Import Meters from the same file". Note this may take several seconds to appear, and several seconds to prompt the next modal screen.
Screen Shot 2022-03-08 at 12 34 33 PM
- Ensure there are proposed counts for meter counts and unlinkable properties. 
- Click confirm 
- Ensure there is an imported meter count.

Test file: [ESPM_JCC_Custom_Download_BETTERTest.xlsx](https://github.com/SEED-platform/seed/files/8209091/ESPM_JCC_Custom_Download_BETTERTest.xlsx)

Test GreenButton:
Inventory > Property detail (one that has meters) > Meters > Upload GreenButton Data > same process as above.
Test File: /seed/tests/data/example-GreenButton-data.xml

#### What are the relevant tickets?
#3127 

#### Screenshots (if appropriate)
Proposed Meter Imports
Screen Shot 2022-03-04 at 3 01 56 PM

Imported Meters
Screen Shot 2022-03-04 at 3 19 51 PM

### CODE REVIEW

**Comment by RDmitchell** [comment]
@perryr16 -- looks great !!

**Comment by Ryoken** [comment]
I am not getting the new divs to show up:

Screen Shot 2022-03-09 at 11 53 45 AM
Screen Shot 2022-03-09 at 12 10 52 PM

File: seed/static/seed/js/controllers/data_upload_modal_controller.js [hunk 1]
```diff
@@ -212,6 +212,13 @@ angular.module('BE.seed.controller.data_upload_modal', [])
       };
 
       var present_parsed_meters_confirmation = function (result) {
+        $scope.proposed_meters_count = result.proposed_imports.length
```
Comment (Ryoken) [inline seed/static/seed/js/controllers/data_upload_modal_controller.js:215]: Could these lines get some semicolons? =D

File: seed/static/seed/partials/data_upload_modal.html [hunk 2]
```diff
@@ -358,6 +358,9 @@ <h4 class="modal-title" ng-switch-when="16" translate>PM_METER_IMPORT_RESULTS</h
                 <div class="alert alert-warning" ng-if="proposed_imports_options.data.length">
                     <div ng-if="proposed_imports_options.data.length">
                         <div translate><b>METER_READING_COUNTS_TITLE</b></div>
+                        <ul class="meter-count" nf-if="proposed_properties_count">
```
Comment (Ryoken) [inline seed/static/seed/partials/data_upload_modal.html]: is nf-if a valid tag or should it be ng-if

File: seed/static/seed/partials/green_button_upload_modal.html [hunk 3]
```diff
@@ -22,6 +22,9 @@ <h4 class="modal-title" ng-switch-when="4" translate>UPLOAD_COMPLETE_TITLE</h4>
             <div class="alert alert-warning green-button-preview">
                 <div>
                     <div translate><b>METER_READING_COUNTS_TITLE</b></div>
+                    <ul class="meter-count" nf-if="proposed_meters_count">
```
Comment (Ryoken) [inline seed/static/seed/partials/green_button_upload_modal.html:25]: nf-if again

**Pull request merged**